includes/admin/payments/view-order-details.php

```diff
@@ -311,12 +311,11 @@
 									<div class="edd-admin-box">
 
 										<div class="edd-admin-box-inside">
-
 											<p>
 												<a href="<?php echo admin_url( '/edit.php?post_type=download&page=edd-reports&tab=logs&payment=' . $payment_id ); ?>"><?php _e( 'View file download log for purchase', 'easy-digital-downloads' ); ?></a>
 											</p>
 											<p>
-												<?php $download_log_url = admin_url( 'edit.php?post_type=download&page=edd-reports&tab=logs&user=' . $user_id ); ?>
+												<?php $download_log_url = admin_url( 'edit.php?post_type=download&page=edd-reports&tab=logs&user='.$user_id.'&customer='.$customer->id ); ?>
```

Comment:
We should just remove the `user_id` parameter here all together, it isn't used anymore.